desk/mar/provider-state.hoon

```diff
@@ -0,0 +1,12 @@
+/-  n=notify
```

Comment:
if we're only ever using these as nouns we should probably just use the noun mark